test/core/dataProvider/proxy/ajax/test.js

```diff
@@ -110,6 +110,7 @@ define([
             })
             .catch(function(err) {
                 assert.ok(false, 'The promise should not be rejected');
+                // eslint-disable-next-line
```

Comment:
Note for future: you can also put `/* eslint-disable no-console */` at the top of such files, if it's too repetitive.

Reply:
Good point! I should have think of it instead of looking at my fingers 😄